core/templates/dev/head/pages/skill-editor-page/modal-templates/add-misconception-modal.directive.html

```diff
@@ -22,9 +22,9 @@ <h3>Add Misconception</h3>
           <span class="oppia-editor-description">Use a name that will be easy for question creators to understand.</span>
         </div>
         <div>
-          <input class="form-control protractor-test-misconception-name-field" type="text" ng-model="misconceptionName" maxlength="30" autofocus>
-          <span ng-if="misconceptionName.length === 30" class="help-block" style="color: red; font-size: smaller">
-            <em>Please use at most 30 characters.</em>
+          <input class="form-control protractor-test-misconception-name-field" type="text" ng-model="misconceptionName" maxlength="50" autofocus>
+          <span ng-if="misconceptionName.length === 50" class="help-block" style="color: red; font-size: smaller">
+            <em>Please use at most 50 characters.</em>
```

Comment:
The PR title says this is increased to 60, but here,it is only 50?

Reply:
Haha! So sorry for that, That's because at first we decided to change the misconception length to 60 but along the way we decided that using 50 chars was better as our already existing validation functions are made to validate string having length upto 60 chars. So that's why. 
I'll change the name of the PR. Thanks!